projects/pydateutil/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
```

Comment:
Just noticed, can you update copyright year?

Reply:
Also in other places

Reply:
done